tests/integration-tests/test_runner.py

```diff
@@ -72,6 +73,7 @@
     "benchmarks": False,
     "benchmarks_target_capacity": 200,
     "benchmarks_max_time": 30,
+    "stackname_suffix": "-" + getpass.getuser(),
```

Comment:
if it doesn't add much value I'd rather leave an empty suffix as default

Reply:
ok, this will impact the other comments